Content/css/style_index.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  background-image: linear-gradient(to top left, #ff52b3, #7ff5ff);
+}
+
+div.container {
+  max-width: min-content;
+  padding: 25px;
+  margin: 0 auto;
+  background-color: #fff7ff96;
+  border-radius: 15px
+}
+
+div.container h1 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 200;
+  color: #d800ff;
+}
+
+div.container form span {
+  font-size: 15px;
+  color: #d800ff;
+}
+
+div.container form {
+  margin-top: 20px;
+}
+
+div.container form label {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 300;
+  color: black;
+}
+
+div.container form input {
+  border: none;
+  border-bottom: 1px solid;
+  background-color: transparent;
+  padding-top: 10px;
+  margin-bottom: 23px;
+  width: 17rem;
+  font-size: 15px;
+  outline: none;
+}
+
+div.container form input[type="email"]:focus-within, input[type="password"]:focus-within {
```

Comment:
According to the documentation, one line should not exceed more than 80 characters. Also, you can configure your VS code as per drupal VS code configuration you can follow this link [https://www.drupal.org/docs/develop/development-tools/editors-and-ides/configuring-visual-studio-code.](url)